include/circt/Dialect/SV/SVStatements.td

```diff
@@ -913,6 +913,11 @@ def MacroDefOp : SVOp<"macro.def",
   }];
 }
 
+def IncludeOp : SVOp<"include"> {
+  let summary = "preprocessor `include directive";
+  let arguments = (ins StrAttr:$target);
+  let assemblyFormat = [{ $target attr-dict }];
+}
```

Comment:
I was going to suggest that the parent should be an MLIR moduleOp. However, the SystemVerilog spec allows this to be used anywhere in the text. 😬 This is correct to allow it anywhere.

Reply:
A related question I had was, what happens with an IncludeOp located in the top-level MLIR module when we are emitting multiple files. I followed the example of MacroDefOp, and have it replicated into each output file.

Reply:
I think replicating makes sense and aligns with other ops.

Ideally, we should move everything to use the emit ops which solves this problem as this is then explicit.